.evergreen/compile.sh

```diff
@@ -11,4 +11,4 @@ echo "Compiling java driver with jdk11"
 
 export JAVA_HOME="/opt/java/jdk11"
 ./gradlew -version
-./gradlew -PxmlReports.enabled=true --info -x test clean check jar testClasses docs
+./gradlew -PxmlReports.enabled=true --info -x test -x tckTest clean check jar testClasses docs
```

Comment:
Should I figure out a way to disable these by default? They are a bit slow.

Reply:
Yes - these only need to be run once per build.

Reply:
Yes - these only need to be run once per build.